apps/neoscan/lib/neoscan/transactions/transactions.ex

```diff
@@ -82,11 +91,11 @@ defmodule Neoscan.Transactions do
   def paginate_transactions(pag, _) do
     transaction_query =
       from(
-        e in Transaction,
+        t in Transaction,
         order_by: [
-          desc: e.id
+          desc: t.block_index
```

Comment:
this will result in transactions being randomly sorted inside the same block. Still I'm not sure if the order of transactions inside the block matters, but sorting by inserted_at might be a solution.

Reply:
Yes actually i will use two column, block_index and array_index. The whole thing is still work in progress. I opened a pull request because I am used to the gitlab flow, but this is not ready for review yet :)